### PR TITLE
Add command line script

### PR DESCRIPTION
**Pull request by liquorvicar**

### CODE REVIEW

File: bin/php-json-schema-generate [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+use Elsevier\JSONSchemaPHPGenerator\Console\GenerateCommand;
+use Symfony\Component\Console\Application;
+
+$application = new Application();
+
+$command = new GenerateCommand();
+$application->add($command);
+$application->setDefaultCommand($command->getName(), true);
+
+$application->run();
```
Comment (ltatakis) [inline bin/php-json-schema-generate]: I am trying to run it as ```./bin/php-json-schema-generate test.json MyFantasticSchema``` with test input being```  "IntegerProperty": {
    "properties": {
      "fooBarTest": {"type": "number"}
    }
  }
}}``` - but no output or errors are generated.

Comment (ltatakis) [reply bin/php-json-schema-generate]: Version of PhP used in 7.0.15

Comment (liquorvicar) [reply bin/php-json-schema-generate]: Ok, as discussed it was failing for you as you didn't have the right deps installed and you had `display_errors` set to Off in your php.ini. So going to merge.

Comment (johnwards) [reply bin/php-json-schema-generate]: Probably needs a readme

Comment (liquorvicar) [reply bin/php-json-schema-generate]: Not sure a README will cover it. The issues @ltatakis had were due to composer and PHP itself.

**Pull request merged**